app/models/metric/ci_mixin/processing.rb

```diff
@@ -14,70 +14,71 @@ def perf_process(interval_name, start_time, end_time, counters_data)
     _log.info("#{log_header} Processing for #{log_specific_targets(resources)}, for range [#{start_time} - #{end_time}]...")
 
     _dummy, t = Benchmark.realtime_block(:total_time) do
-      # Take the raw metrics and create hashes out of them
-      rt_rows = {}
-
-      counters_data.each do |resource, data|
-        process_counter_values(resource, data, interval_name, rt_rows) ## rt_rows changes
-      end
-
+      rt_rows = convert_raw_metrics_to_hashes(counters_data, interval_name)
       parameters = convert_metrics(resources, interval_name, start_time, end_time, rt_rows)
       write_metrics(parameters)
-      update_and_publish_metrics(interval_orig, start_time, end_time, resources, rt_rows)
+      update_last_perf_capture_on(resources, end_time)
+      raise_perf_complete_alerts(resources)
+      rollup_to_parent_metrics(resources, interval_orig, start_time, end_time)
+      publish_metrics(rt_rows) if syndicate_metrics?
     end
     _log.info("#{log_header} Processing for #{log_specific_targets(resources)}, for range [#{start_time} - #{end_time}]...Complete - Timings: #{t.inspect}")
   end
 
-  def process_counter_values(resource, data, interval_name, rt_rows)
-    log_header = "[#{interval_name}]"
+  def convert_raw_metrics_to_hashes(counters_data, interval_name)
```

Comment:
If you are changing the method name, perhaps you should change the Benchmark title as well, since they were originally made to match.

Reply:
I would prefer to not change this benchmark. I'll change the name back